cloudslang-compiler/src/test/java/io/cloudslang/lang/compiler/LoadSystemPropertiesTest.java

```diff
@@ -183,6 +210,19 @@ public void testInvalidCharsNamespace() throws Exception {
     }
 
     @Test
+    public void testInvalidCharsNamespaceFromSource() throws Exception {
```

Comment:
IMHO you should validate that you get all the exceptions when having multiple ones